Trackers/centroid_tracker/centroid.py

```diff
@@ -1,30 +1,112 @@
 import numpy as np
+from requests import delete
 from ..track import Track
+from scipy.spatial import distance as dist
 
 class centroid_track(Track):
-    def __init__(self, id, bbox, centroid):
+    def __init__(self, id, bbox, centroid, hits, miss):
         self.centroid = centroid
-        Track.__init__(self, id, bbox)
+        Track.__init__(self, id, bbox, hits, miss)
 
 class Centroid_tracker():
     def __init__(self):
         self.tracks = []
+        self.nextID=0
         self.max_age = 10
         self.min_hits = 3
         self.thres_distance = 20
 
     def add_track(self, id, bbox, centroid):
         """Create a centroid track object and adds to tracks list
         """
-        pass
+
+        '''
+        - add centroid to track list
+        - id++
+        - add instance from class track
+        - if hits > min_hits add track
+        '''
```

Comment:
Provide proper documentation of the function including its parameters and return type etc